### ISSUE TITLE
Profile refresh in the Admin UI should surface avatar/header size errors

### ISSUE THREAD

**Issue by thegcat** [mastodon/mastodon#31388]
### Steps to reproduce the problem

1. Klick Refresh Profile in the Admin UI for a remote account
2. The UI shows a success banner
3. The account has missing avatar and banner

### Expected behaviour

Show an error

### Actual behaviour

Show success

### Detailed description

We found out that the issue was with the remote avatar and banner images, those were over 2MiB in size, which Mastodon doesn't process. I can understand the limit, but at least the admin UI when explicitly trying to refresh the account should surface those as errors as in the end the remote avatar and banner were not saved locally. This also applies to the `tootctl` interface to refresh accounts.

### Mastodon instance

toot.kif.rocks

### Mastodon version

v4.2.10

### Browser name and version

Irrelevant

### Operating system

Irrelevant

**Comment by ThisIsMissEm** [comment]
The refresh here happens asynchronously, and would require storing the error in the database somewhere, so there isn't really currently a good way to surface meaningful information back, afaik.

**Comment by thegcat** [comment]
As far as I could follow the callstack for `Admin::AccountsController#redownload` it looked synchronous, except when there‘s a network error or something like that, in which case it gets retried in the background. It also seemed when we fixed the problem why it didn‘t work that it was already finished when the account page was re-shown, as the avatar and header were shown correctly.

In any case, if it is asynchronous, at least the verbiage of the success message should be updated to indicate that the update was not successfully performed but successfully queued (`redownloaded_msg: Successfully refreshed %{username}'s profile from origin`).

**Comment by ThisIsMissEm** [comment]
Yeah, I think there's too many degrees of separation between `Admin::AccountsController#redownload` and where the errors may occur that there's simply no good way to surface those back to the user. In the case of the media for the account, the download actually happens asynchronously: `RedownloadAvatarWorker.perform_in` and `RedownloadHeaderWorker.perform_in` in `ActivityPub::ProcessAccountService`

The download actually happens via `app/models/concerns/remotable.rb`, which is where the size is validated with the Account model is saved from `RedownloadHeaderWorker` and `RedownloadAvatarWorker`
